File: iosApp/iosApp/Presentation/Auth/SignIn/SignInUiState.swift
//
//  SignInUiState.swift
//  ConCafe
//
//  Created by 홍희표 on 2026/03/07.
//

import Foundation

struct SignInUiState {
    var email: String = ""
    
    var password: String = ""
    
    var isLoading: Bool = false
    
    var errorMessage: String?
    
    static let empty = SignInUiState()
}

File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/auth/signin/SignInUiState.kt
package com.hhp227.concafe.presentation.auth.signin

data class SignInUiState(
    val email: String = "",
    val password: String = "",
    val isLoading: Boolean = false,
    val errorMessage: String? = null
) {
    companion object {
        fun empty() = SignInUiState()
    }
}
